sdk/storage/Azure.Storage.Blobs/perf/Azure.Storage.Blobs.Perf/Options/PartitionedTransferOptions.cs

```diff
@@ -50,6 +59,9 @@ public class PartitionedTransferOptions : SizeOptions, IBlobClientOptionsProvide
         [Option("client-encryption")]
         public string EncryptionVersionString { get; set; }
 
+        [Option("cache-responses")]
+        public bool CacheResponses { get; set; }
```

Comment:
Curious: would this need to be exposed in a public APIs for customers to get the perf benefit of this, or is it just an improvement to boost perf on our own perf tests?

Reply:
This whole type must be redone, I'll remove it from this PR.

Reply:
Change is undone